dev/ci/ci-basic-overlay.sh

```diff
@@ -7,138 +7,138 @@
 ########################################################################
 # MathComp
 ########################################################################
-: ${mathcomp_CI_BRANCH:=master}
-: ${mathcomp_CI_GITURL:=https://github.com/math-comp/math-comp.git}
+mathcomp_CI_BRANCH=master
+mathcomp_CI_GITURL=https://github.com/math-comp/math-comp.git
```

Comment:
Why is the old solution problematic? Can you still set these variables from `make` or user env?

Reply:
I agree with emilio.

You may want to user `true` rather than `:` or define something line `default { true; }..... default ${VAR:=def}`

Reply:
OK I put the default value assignment back.
`:` is idiomatic for this so I left it.